Modules/EMCAL/src/ClusterTask.cxx

```diff
@@ -113,36 +137,59 @@ void ClusterTask::startOfCycle()
 
 void ClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
+  std::string InputCell = "emcal-cells";
+  std::string InputCellTR = "emcal-cellstriggerecords";
 
-  std::string InputCell, InputCluster, InputCellIndex;
-  std::string InputCellTrig, InputClusterTrig, InputCellIndexTrig;
+  auto Cell = ctx.inputs().get<gsl::span<o2::emcal::Cell>>(InputCell.c_str());
+  auto CellTR = ctx.inputs().get<gsl::span<o2::emcal::TriggerRecord>>(InputCellTR.c_str());
 
-  InputCell = "emcal-cells";
-  InputCellTrig = "emcal-cellstriggerecords";
+  if (mInternalClusterizer) {
+    std::vector<o2::emcal::Cluster> Cluster;
+    std::vector<int> CellIndex;
+    std::vector<o2::emcal::TriggerRecord> ClusterTR, CellIndexTR;
 
-  InputCluster = "emcal-clusters";
-  InputClusterTrig = "emcal-clustertriggerecords";
+    findClustersInternal(Cell, CellTR, Cluster, ClusterTR, CellIndex, CellIndexTR); //do internal clustering here
+    ILOG(Info, Support) << "Found  " << Cluster.size() << " CLusters  " << ENDM;
+    ILOG(Info, Support) << "Found  " << ClusterTR.size() << " ClusterTR  " << ENDM;
+    ILOG(Info, Support) << "Found  " << CellIndex.size() << " CellIndex  " << ENDM;
+    ILOG(Info, Support) << "Found  " << CellIndexTR.size() << " CellIndexTR  " << ENDM;
```

Comment:
Change to Debug. Clusters.size() and ClusterTR.size() should also be monitored in a histogram in analyseTimeframe.

Reply:
The same comment is repeated here?